sdk/spring/spring-cloud-azure-autoconfigure/src/test/java/com/azure/spring/cloud/autoconfigure/useragent/http/AppConfigurationUserAgentTests.java

```diff
@@ -22,17 +22,20 @@
 import org.springframework.boot.test.system.OutputCaptureExtension;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
 /**
- * Disabled because  for ConfigurationClient, the header is not outputted in log.
- * I debugged and confirmed that User-Agent been set successfully.
- * The reason why the User-Agent not outputted in log is not clear.
+ * Disabled because  for ConfigurationClient, the header is not outputted in log. I debugged and confirmed that
+ * User-Agent been set successfully. The reason why the User-Agent not outputted in log is not clear.
  */
 @Disabled
 @Isolated("Run this by itself as it captures System.out")
 @ExtendWith(OutputCaptureExtension.class)
 public class AppConfigurationUserAgentTests {
 
+    String format1 = String.format("User-Agent:%s", AzureSpringIdentifier.AZURE_SPRING_APP_CONFIG);
+    String format2 = String.format("\"User-Agent\":\"%s", AzureSpringIdentifier.AZURE_SPRING_APP_CONFIG);
```

Comment:
no need to be class field, it's fine to put it in the method, otherwise we will need something like private static final String